cms/core/checks.py

```diff
@@ -50,3 +52,27 @@ def check_wagtail_pages(*args: Any, **kwargs: Any) -> Iterator[CheckMessage]:
                     " or a subclass of that class.",
                     obj=model,
                 )
+
+
+@register()
+def check_aws_iam_credentials(*args: Any, **kwargs: Any) -> list[Error]:
+    """Check that required IAM credentials are present when running in AWS."""
+    errors: list[Error] = []
+
+    env = os.environ.copy()
+
+    if "AWS_REGION" not in env:
+        return errors
+
+    aws_credentials_settings = ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY"]
```

Comment:
Issue: This isn't how authentication works in production. It uses service accounts in Kubernetes, rather than fixed credentials. That means this check will always fail. I'd suggest removing it.

Reply:
Removed in [8705344](https://github.com/ONSdigital/dis-wagtail/pull/210/commits/8705344fbc1d42fabe07e743de5341c74b47a851)

Shouldn't the following comment also be changed? 🤔 
https://github.com/ONSdigital/dis-wagtail/blob/ed64a48bae7621c9b2a319e63134d9d18b2a73d8/cms/settings/base.py#L427-L432